plugins/modules/aws_codebuild.py

```diff
@@ -275,18 +301,34 @@
       type: int
       sample: 60
     tags:
-      description: Tags added to the project
+      description:
+        - Tags added to the project in the boto3 list of dictionaries format.
+        - I(tags) and I(reource_tags) represent the same information in
+          different formats.
       returned: when configured
       type: list
+    reource_tags:
+      description:
+        - A simple dictionary representing the tags added to the project.
+        - I(tags) and I(reource_tags) represent the same information in
+          different formats.
+      returned: when configured
+      type: dict
+      version_added: 4.0.0
```

Comment:
@markuman @tremble But what about these params here? version_added:  is 4.0.0.

Reply:
ups

Reply:
I'll update that and backport if/when this is merged.